### DISCUSSION TITLE
Is it possible to have the step value on NumberField not effect the fraction rounding precision?

### DISCUSSION THREAD

**Discussion by timmaier** [unovue/reka-ui#977] [Q&A]
When I set the step prop to 1 I still want to be able to enter fractions for example 10.35 via the input itself when not using the NumberFieldIncrement and NumberFieldDecrement component.

**Comment by zernonia** [comment]
Hey @timmaier , by right it should persist the decimal value, and increase/decrease based on the steps. eg: 18.5 -> 19.5 
If it doesn't do this let's create an issue for this 😁

**Comment by timmaier** [comment]
Hey @zernonia thanks for the prompt response. 

Here's a stack blitz showing my usecase. Maybe it has something to do with my computed shadow value. I would expect I could enter 1.1, 1.2, 1.3 etc as a value and it persist. However doing so will always round down or up to whatever I set step too.

https://stackblitz.com/edit/ggqkok?file=src%2FApp.vue
